### PR TITLE
Added ReportUnit Alias

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by devlead** [comment]
@gep13 need to be refactored to use `Cake.Core.Tooling.Tool<TToolSettings>`

**Comment by gep13** [comment]
>@devlead said....
@gep13 need to be refactored to use Cake.Core.Tooling.Tool<TToolSettings>

Yip, I rebased and pushed all the PR's earlier today, working on fixing them just now

@patriksvensson @devlead would you be able to review the refactoring that I have done in the new commit, and if you are both happy, I can squash it.

Thanks!

File: src/Cake.Common.Tests/Fixtures/Tools/ReportUnit/ReportUnitDirectoryFixture.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using Cake.Common.Tools.ReportUnit;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tests.Fixtures.Tools.ReportUnit
+{
+    class ReportUnitDirectoryFixture : ToolFixture<ReportUnitSettings>
```
Comment (patriksvensson) [inline src/Cake.Common.Tests/Fixtures/Tools/ReportUnit/ReportUnitDirectoryFixture.cs]: Should have explicit access modifier (internal).

**Comment by patriksvensson** [comment]
Left comment. Other than that this looks good to me :+1:

**Comment by gep13** [comment]
Good catch, updated and rebased this.

**Comment by devlead** [comment]
Looks great @gep13 :+1: , could you just rebase into 1 commit.

**Comment by gep13** [comment]
@devlead done :smile:

**Comment by devlead** [comment]
@gep13 thanks:+1:

**Pull request merged**